sleap_nn/data/streaming_datasets.py

```
@@ -0,0 +1,380 @@
+"""Custom `litdata.StreamingDataset`s for different models."""
+
+from kornia.geometry.transform import crop_and_resize
+from omegaconf import DictConfig
+from typing import List, Optional, Tuple
```

Ruff

`typing.List` imported but unused

Remove unused import

(F401)